src/pages/CommitQueue.tsx

```diff
@@ -37,19 +39,32 @@ export const CommitQueue: React.VFC = () => {
 
   return (
     <PageWrapper>
-      <PageTitle
-        pageTitle={`Commit Queue - ${projectIdentifier}`}
-        title="Commit Queue"
-        badge={
-          <Badge variant="darkgray">
-            {buildBadgeString(queue ? queue.length : 0)}
-          </Badge>
-        }
-        loading={loading}
-      />
-      {commitQueue?.message && (
-        <Body data-cy="commit-queue-message">{commitQueue.message}</Body>
-      )}
+      <PageHeader>
+        <Column>
+          <PageTitle
+            pageTitle={`Commit Queue - ${projectIdentifier}`}
+            title="Commit Queue"
+            badge={
+              <Badge variant="darkgray">
+                {buildBadgeString(queue ? queue.length : 0)}
+              </Badge>
+            }
+            loading={loading}
+            size="large"
+          />
+          {commitQueue?.message && (
+            <Body data-cy="commit-queue-message">{commitQueue.message}</Body>
+          )}
```

Comment:
I'm thinking that this should be removed from the `<PageHeader />` and instead be placed directly above the `<HR />`. Then you could also delete the `Column` style

Reply:
<img width="1174" alt="image" src="https://user-images.githubusercontent.com/4605522/215105867-9c8a85f8-d22f-493b-ac38-4712561b5d8c.png">
Done